csharp/src/AddressBook/Addressbook.cs

```diff
@@ -77,9 +77,18 @@ public sealed partial class Person : pb::IMessage<Person>, global::System.IEquat
     }
 
     public Person() { }
+
     public Person(Person other) {
```

Comment:
What's the rationale behind making the constructor cloning by default? Is it to prevent users from creating messages with aliased fields by mistake?